docs/examples/ex_extract.rst

```diff
@@ -14,11 +14,145 @@ modeled values.
 Regular Grid Models
 -------------------
 
-:py:func:`pysatModels.utils.extract.extract_modeled_observations` supports
-extracting values from models on a regular grid. The function can linearly
-interpolate model values onto instrument locations or use the nearest modeled
-location. Uses :py:func:`scipy.interpolate.interpn` as the underlying
-interpolation function.
+:py:func:`pysatModels.utils.extract.extract_modelled_observations` supports
+extracting values from models on a regular grid onto observed locations. The
+function can linearly interpolate model values onto instrument locations or use
+the nearest modeled location. Uses :py:func:`scipy.interpolate.interpn` as the
+underlying interpolation function. This function can handle either
+:py:mod:`pandas` or :py:mod:`xarray` formatted :py:class:`pysat.Instrument`
+observational data.  Because multi-dimensional data can be more complicated,
+let's use `Jicamarca ISR drift data <https://pysatmadrigal.readthedocs.io/en/latest/supported_instruments.html#jro-isr>`_ as an example.
+
+.. code:: python
+
+   import datetime as dt
+   import pandas as pds
+   import pysat
+   from pysatMadrigal.instruments import jro_isr
+   import pysatModels
+
+   stime = dt.datetime(2021, 1, 3)
+   jro = pysat.Instrument(inst_module=jro_isr, tag='drifts', user='Your Name',
+                          password='your.email@inst.type')
+
+    # Download data if necessary and then load it
+    if stime not in jro.files.files:
+        jro.download(start=stime)
+    jro.load(date=stime)
+
+    # Get fake model data from the pysat model test instrument
+    mod_drange = pds.date_range(stime, stime + dt.timedelta(days=1), freq='1D')
+    model = pysat.Instrument('pysat', 'testmodel', tag='',
+                             file_date_range=mod_drange)
+    model.load(date=stime)
+
+    # Check the loaded variables
+    print(jro.variables, model.variables)
+
```

Comment:
Is this warning expected from this command? If so, suggest adding a note for the user.
 
```
pysat WARNING: unknown data variable(s) ['vipn2' 'dvipn2' 'vipe1' 'dvipe1' 'vi72' 'dvi72' 'vi82' 'dvi82'], using only: ['nwlos' 'range' 'vipn' 'dvipn' 'vipe' 'dvipe' 'vi7' 'dvi7' 'vi8' 'dvi8'
 'paiwl' 'pacwl' 'pbiwl' 'pbcwl' 'pciel' 'pccel' 'pdiel' 'pdcel' 'jro10'
 'jro11']
```